modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/tree/BPlusTree.java

```diff
@@ -1497,7 +1497,7 @@ public final <R> R findOne(L row, Object x) throws IgniteCheckedException {
      * @return Found result or {@code null}.
      * @throws IgniteCheckedException If failed.
      */
-    public final <R> R findOne(L row, TreeRowClosure<L, T> c, Object x) throws IgniteCheckedException {
+    public <R> R findOne(L row, TreeRowClosure<L, T> c, Object x) throws IgniteCheckedException {
```

Comment:
seems we needn`t touch this code, probably the right way is to fix the reporting code and adopt it for test usage, what do you think ?

Reply:
I updated the "reporting" code and redesigned the test to use a log listener.
Thanks.